fe/fe-core/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -526,11 +530,23 @@ public synchronized void addTableIndexes(OlapTable table) {
 
     @Override
     public String toString() {
+        Database db = Catalog.getCurrentCatalog().getDb(dbId);
+        String dbName = null;
+        String tableNameList = null;
+        if (db != null) {
+           dbName = db.getFullName();
+           tableNameList = tableIdList.stream()
+                   .map(db::getTable)
+                   .filter(Objects::nonNull)
+                   .map(Table::getName).collect(Collectors.joining(","));
+        }
```

Comment:
We have add `show database id` stmt to find database name from database id.
So there is no need to print database name here.
And a database may have thousands of tables, so print all table here may be disaster.

Reply:
This PR #5807 may solve your problem